### ISSUE TITLE
Is there a way to keep itemid using DataRows to import data.

### ISSUE THREAD

**Issue by iiunknown** [SharePoint/PnP-Sites-Core#2490]
#### Category
[ X ] Bug
[  ] Enhancement

#### Environment
[ X ] Office 365 / SharePoint Online
[ ] SharePoint 2016
[ ] SharePoint 2013

#### Expected or Desired Behavior

In termplate.xml file, there are many DataRows including ID DataValue.  Do Apply-PnPProvisioningTemplate command, the data is not imported into the list.

#### Observed Behavior

The listitem is created and keep ItemId.

**Comment by SchauDK** [comment]
> **Observed Behavior**
> The listitem is created and keep ItemId.

Do you mean that this is the desired behaviour?

That's not possible. The ItemIds are maintained in a table within the content database.

**Comment by iiunknown** [comment]
> > **Observed Behavior**
> > The listitem is created and keep ItemId.
> 
> Do you mean that this is the desired behaviour?
> 
> That's not possible. The ItemIds are maintained in a table within the content database.

Yes, I want to find a way. In many scenarios, I create some items, delete some items, and then create some items again. If there are other List with this Lookup relationship, then the Lookup relationship will be lost after exporting and importing data.

**Comment by SchauDK** [comment]
You'll always get new ItemIds when creating new list items.
You need to workaround it using extensibility handlers and map the old ids to the new ids and then update your lookup fields.
